### ISSUE TITLE
Miranda's Feedback: Chart Improvments

### ISSUE THREAD

**Issue by lovelysandlonelys** [conrad-blucher-institute/semaphore#494]
Chart Improvements

- Gridlines: Great addition!
- Font Change: Switching from Aptos to Arial I think would give a cleaner and more professional look.
- X-Axis Abbreviations: Using abbreviations for the days (e.g., Mon, Tue) simplifies the x-axis and improves readability. Additionally, removing the bold style from the x-axis text will make it look more streamlined.
- Prediction Lines: Adjusting prediction lines from dashes to dots will enhance the visibility of rapid temperature changes and make them more distinct. This will also improve differentiation in the legend, as the current observation and prediction lines look too similar.
- Color Adjustments: While I like the similarity between prediction and observation line colors, could we make the dark blue line a bit of a lighter shade? This would better contrast air and water temperature data and improve clarity.

Change the line colors back to the way they are displayed on TPW

**Comment by lovelysandlonelys** [comment]
Mostly done, we're being blocked by Dr. Tissot for replacing the banner which will become a separate task.
